diff -urpN linux-3.14.1/arch/x86/syscalls/syscall_64.tbl linux-3.14.1-m/arch/x86/syscalls/syscall_64.tbl
--- linux-3.14.1/arch/x86/syscalls/syscall_64.tbl	2014-04-14 15:50:10.000000000 +0200
+++ linux-3.14.1-m/arch/x86/syscalls/syscall_64.tbl	2017-10-25 14:53:42.608388838 +0200
@@ -322,6 +322,7 @@
 313	common	finit_module		sys_finit_module
 314	common	sched_setattr		sys_sched_setattr
 315	common	sched_getattr		sys_sched_getattr
+316	common	lin_hello		sys_lin_hello
 
 #
 # x32-specific system call numbers start at 512 to avoid cache impact
Los ficheros binarios linux-3.14.1/arch/x86/syscalls/.syscall_64.tbl.swp y linux-3.14.1-m/arch/x86/syscalls/.syscall_64.tbl.swp son distintos
diff -urpN linux-3.14.1/kernel/hello.c linux-3.14.1-m/kernel/hello.c
--- linux-3.14.1/kernel/hello.c	1970-01-01 01:00:00.000000000 +0100
+++ linux-3.14.1-m/kernel/hello.c	2017-10-25 14:49:29.177729479 +0200
@@ -0,0 +1,7 @@
+#include <linux/syscalls.h> /* For SYSCALL_DEFINEi() */
+#include <linux/kernel.h>
+SYSCALL_DEFINE0(lin_hello)
+{
+printk(KERN_DEBUG "Hello world!\n");
+return 0;
+}
diff -urpN linux-3.14.1/kernel/Makefile linux-3.14.1-m/kernel/Makefile
--- linux-3.14.1/kernel/Makefile	2014-04-14 15:50:10.000000000 +0200
+++ linux-3.14.1-m/kernel/Makefile	2017-10-25 14:54:14.732442654 +0200
@@ -10,7 +10,7 @@ obj-y     = fork.o exec_domain.o panic.o
 	    kthread.o sys_ni.o posix-cpu-timers.o \
 	    hrtimer.o nsproxy.o \
 	    notifier.o ksysfs.o cred.o reboot.o \
-	    async.o range.o groups.o smpboot.o
+	    async.o range.o groups.o smpboot.o hello.c
 
 ifdef CONFIG_FUNCTION_TRACER
 # Do not trace debug files and internal ftrace files
